src/openrct2/ride/Track.cpp

```diff
@@ -35,276 +36,277 @@
 #include "ride_ratings.h"
 #include "RideGroupManager.h"
 #include "station.h"
-#include "track.h"
+#include "Track.h"
 #include "track_data.h"
-#include "../world/map.h"
```

Comment:
Includes should be sorted

Reply:
It is. Haven't you looked a few lines above?!

Reply:
All of them. `footpath.h`, `Track.h`, `RideGroupManager.h` are misplaced

Reply:
How is Track.h misplaced?! Track should be before track_data.

Reply:
Oh, in case-insensitive sorting that might be true. Fair enough.

Reply:
`footpath` still has to be moved

Reply:
Has been fixed.